bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java

```diff
@@ -636,16 +636,12 @@ public void onException(RetriesExhaustedWithDetailsException exception,
 
     protected void logExceptions(final Context context,
         RetriesExhaustedWithDetailsException exception) {
-      List<Throwable> causes = exception.getCauses();
-      String failEventId = UUID.randomUUID().toString();
-      String message =
-          String.format("For context %s: eventId: %s occured during bulk writing.", context,
-            failEventId);
-      LOG.warn(message, exception);
-      for (int i = 0; i < causes.size(); i++) {
-        LOG.warn(String.format("context %s: eventId %s excepition %d", context, failEventId, i),
-          causes.get(i));
-      }
+      // RetriesExhaustedWithDetailsException has its pre-JDK1.7 semantics
```

Comment:
Hard to understand this comment, can you rewrite it in simpler way? Focus on the difference between #getCause() and #getExhaustiveDescription().